src/Rules/File/FileExtensionRule.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigCsFixer\Rules\File;
+
+use TwigCsFixer\File\FileHelper;
+use TwigCsFixer\Rules\AbstractRule;
+use TwigCsFixer\Rules\ConfigurableRuleInterface;
+
+/**
+ * Ensures file name uses two extensions (e.g. index.html.twig).
+ */
+final class FileExtensionRule extends AbstractRule implements ConfigurableRuleInterface
+{
+    /**
+     * @param array<string> $ignoredSubDirectories
+     */
+    public function __construct(
+        private ?string $baseDirectory = null,
+        private array $ignoredSubDirectories = [],
```

Comment:
For the FileNameRule and the DirectoryNameRule, I needed:
- A base directory to start validating the path at **/templates/, and not directory before
- A ignoredSubDirectories because the `templates/bundles` doesn't follow the same convention https://symfony.com/doc/current/controller/error_pages.html

But here, I'm not sure we need the same configuration.
WDYT about starting without @smnandre ?
We can still add the config later if needed.

You'll just call
```
$fileName = FileHelper::getFileName(
    $token->getFilename(),
);
```

Reply:
Of course ! And thx for the explanations :)